src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int max = 100;
+
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randomnumber = new Random().nextInt(max);

```

Comment:
ColorSupplier and Random should be class level field
```suggestion
        ColorSupplier colorSupplier = new ColorSupplier();
        int randomnumber = new Random().nextInt(max);

```